### PR TITLE
Add migration code to merge unmerged measurements

### LINKED ISSUE #4556: Clean up unmerged measurements
Clean up the unmerged measurements in the database caused by #4554.

### PR DESCRIPTION
**Pull request by fniessink**
Fixes #4556.

### CODE REVIEW

File: components/internal_server/src/quality_time_server.py [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
 
-from initialization import init_bottle  # skipcq: FLK-E402
+from initialization import init_bottle, merge_unmerged_measurements  # skipcq: FLK-E402
 
 
 def serve() -> None:  # pragma: no cover-behave
     """Connect to the database and start the application server."""
     logging.getLogger().setLevel(logging.INFO)
     database = init_database()
+    merge_unmerged_measurements(database)
```
Comment (wkoot) [inline components/internal_server/src/quality_time_server.py:23]: Do we have a followup story / reminder to remove this in the next release?

**Review by wkoot — APPROVED** [review]

File: components/internal_server/src/quality_time_server.py (hunk 1 above)
Comment (fniessink) [reply components/internal_server/src/quality_time_server.py:23]: That would mean forcing people to install the upcoming release and not skip it. If the code is not too slow it doesn't harm to leave it in until the next major release.

**Review by wkoot — APPROVED** [review]

File: components/internal_server/src/quality_time_server.py (hunk 1 above)
Comment (eric-nieuwland) [inline components/internal_server/src/quality_time_server.py:23]: Why not register in the database that this code has run?
This call could then be made conditional to absence of that registration.

Comment (fniessink) [reply components/internal_server/src/quality_time_server.py:23]: Good question. The advantage is that the code does not run multiple times. The disadvantage is that we need to set a flag which pollutes the database and/or would need to be removed at a later time. 

The migration only runs on startup and when there is nothing to do, should be pretty quick and not write anything to the database. If it takes more than a few seconds on our biggest database, we'll consider adding the flag.

**Pull request merged**